src/ncp/example_vendor_hook.cpp

```diff
@@ -52,6 +53,69 @@ otError NcpBase::VendorCommandHandler(uint8_t aHeader, unsigned int aCommand)
     return error;
 }
 
+void NcpBase::VendorHandleFrameRemovedFromNcpBuffer(NcpFrameBuffer::FrameTag aFrameTag)
+{
+    // This method is a callback which mirrors `NcpBase::HandleFrameRemovedFromNcpBuffer()`.
+    // It is called when a spinel frame is sent and removed from NCP buffer.
+    //
+    // (a) This can be used to track and verify that a vendor spinel frame response is
+    //     delivered to the host (tracking the frame using its tag).
+    //
+    // (b) It indicates that NCP buffer space is now available (since a spinel frame is
+    //     removed). This can be used to implement reliability mechanisms to re-send
+    //     a failed spinel command response (or an async spinel frame) transmission
+    //     (failed earlier due to NCP buffer being full).
+
+    OT_UNUSED_VARIABLE(aFrameTag);
+}
+
+otError VendorGetPropertyHandler(spinel_prop_key_t aPropKey)
+{
+    otError error = OT_ERROR_NONE;
+
+    switch (aPropKey)
+    {
+
+    // TODO: Implement your property get handlers here.
+    //
+    // Get handler should get/retrieve the property value and then encode and write the
+    // value into the NCP buffer. If the "get" operation itself fails, handler should
+    // write a `LAST_STATUS` with the error status into the NCP buffer. `OT_ERROR_NO_BUFS`
+    // should be returned if NCP buffer is full and response cannot be written.
+
+    default:
+        error = OT_ERROR_NOT_FOUND;
+        break;
+    }
+
+    return error;
+}
+
+otError VendorSetPropertyHandler(spinel_prop_key_t aPropKey)
+{
+    otError error = OT_ERROR_NONE;
+
+    switch (aPropKey)
+    {
+
+    // TODO: Implement your property set handlers here.
+    //
+    // Set handler  should first decode/parse the value from the input Spinel frame
```

Comment:
nit: double space between "handler" and "should"
nit: choose either "decode" or "parse"